solhint.config.js

```diff
@@ -0,0 +1,20 @@
+const customRules = require('./scripts/solhint-custom');
+
+const rules = [
+  'no-unused-vars',
+  'const-name-snakecase',
+  'contract-name-camelcase',
+  'event-name-camelcase',
+  'func-name-mixedcase',
+  'func-param-name-mixedcase',
+  'modifier-name-mixedcase',
+  'var-name-mixedcase',
+  'imports-on-top',
+  'no-global-import',
+  ...customRules.map(r => `openzeppelin/${r.ruleId}`),
+];
+
+module.exports = {
+  plugins: ['openzeppelin'],
+  rules: Object.fromEntries(rules.map(r => [r, 'error'])),
```

Comment:
Why not define it like an object?

```
const rules = {
  'no-unused-vars': 'error',
  ...
}
```

Not a blocker because everything shows an "error", I just wanted to point out that the array doesn't seem to be needed

Reply:
We need to use `fromEntries` in some way because we get the names of the custom rules dynamically:

```
customRules.map(r => `openzeppelin/${r.ruleId}`)
```

The rest could be an object, but I had a feeling that this was clearer? Let me know if it's too confusing.